api/v1alpha1/harbor_types.go

```diff
@@ -147,6 +147,10 @@ type RegistryComponent struct {
 
 	// +optional
 	CacheSecret string `json:"cacheSecret,omitempty"`
+
+	// +optional
+	Redirect bool `json:"redirect,omitempty"`
```

Comment:
This will change the default value from true to false

Reply:
yes, the default value is false, and the user can change it by the harbor CR spec.

Reply:
@holyhope 

It seems @soulseen is correct, the previous default value is `false`.

Check the template file `assets/templates/registry/config.yaml`.

```yaml
redirect:	  redirect:
    disable: false
```

Reply:
What does `.Spec.Redirect == true` means? Should `redirect: false` or `redirect: true`?
You are right default value for `.Spec.Redirect` is `false`. But I though that would mean `disable: true`.
What do you think about renaming it `DisableRedirect`?

```suggestion
	DisableRedirect bool `json:"disableRedirect,omitempty"`
```

Reply:
good point.